echo/server/dembrane/config.py

```
@@ -162,6 +162,12 @@
 DISABLE_CORS = os.environ.get("DISABLE_CORS", "false").lower() in ["true", "1"]
 logger.debug(f"DISABLE_CORS: {DISABLE_CORS}")
 
+ENABLE_ENGLISH_TRANSCRIPTION_WITH_LITELLM = os.environ.get(
+    "ENABLE_ENGLISH_TRANSCRIPTION_WITH_LITELLM", "false"
+).lower() in ["true", "1"]
+# ENABLE_ENGLISH_TRANSCRIPTION_WITH_LITELLM is optional and defaults to false
+logger.debug("ENABLE_ENGLISH_TRANSCRIPTION_WITH_LITELLM: %s", ENABLE_ENGLISH_TRANSCRIPTION_WITH_LITELLM)
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)